leshan-client-demo/src/main/java/org/eclipse/leshan/client/demo/LeshanClientDemo.java

```diff
@@ -513,6 +534,19 @@ public static void main(final String[] args) {
             }
         }
 
+        CertificateUsage certificateUsage = CertificateUsage.DOMAIN_ISSUER_CERTIFICATE;
+        if (cl.hasOption("cu")) {
+            certificateUsage = CertificateUsage.fromCode(Integer.parseInt(cl.getOptionValue("cu")));
+            if (trustStore.isEmpty()) {
+                if (certificateUsage == CertificateUsage.SERVICE_CERTIFICATE_CONSTRAINT) {
+                    System.err.println(
+                            "You need to set a truststore when you are using \"service_certificate_constraint\" usage");
```

Comment:
In above help you use `service certificate constraint` and in here `service_certificate_constraint` -- I would recommend using what is in help.

Reply:
I will change this.

Reply:
Done.